cmd/soroban-rpc/internal/methods/simulate_transaction.go

```diff
@@ -97,16 +97,17 @@ type SimulateTransactionCost struct {
 	MemoryBytes     uint64 `json:"memBytes,string"`
 }
 
-type InvokeHostFunctionResult struct {
-	XDR string `json:"xdr"`
+type SimulateTransactionResult struct {
+	Auth      []string `json:"auth"`
+	Footprint string   `json:"footprint"`
+	XDR       string   `json:"xdr"`
 }
 
 type SimulateTransactionResponse struct {
-	Error        string                     `json:"error,omitempty"`
-	Results      []InvokeHostFunctionResult `json:"results,omitempty"`
-	Footprint    string                     `json:"footprint"`
-	Cost         SimulateTransactionCost    `json:"cost"`
-	LatestLedger int64                      `json:"latestLedger,string"`
+	Error        string                      `json:"error,omitempty"`
+	Results      []SimulateTransactionResult `json:"results,omitempty"`
```

Comment:
Right now there can only be one because Core only accepts a host function invocation per transaction.

However, that may change in the future and a slice is used to future-proof it.